lib/data_source/add_task_controller.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:getx_test_1/data_source/add_task_starter.dart';
+import 'package:getx_test_1/data_source/in_memory_task_data_source.dart';
+import 'package:getx_test_1/data_source/task_data_source.dart';
+
+import '../task/models/task_model.dart';
+
+class AddTaskController extends GetxController {
+  final TaskDataSource _dataSource = InMemoryTaskDataSource.instance;
+  final RxList<TaskModel> tasks = RxList();
+  bool submitted = false;
+
+  final TextEditingController newTitle = TextEditingController();
+  final TextEditingController newNote = TextEditingController();
+  final TextEditingController newDate = TextEditingController();
```

Comment:
`GetxController` has a callback function when the controller is disposed. So dispose the TextEditingControllers in this function